From http://dev.gentoo.org/~wesolows

This patch replaces tha panic message advising the user to press L1-A
with one that includes Stop-A since most keyboards are now marked that way.
From Ciaranm.

-------------------------------------------------------------------------------

diff -burN linux-2.6.4-rc2-0/kernel/panic.c linux-2.6.4-rc2-42/kernel/panic.c
--- linux-2.6.4-rc2-0/kernel/panic.c	2004-03-06 15:03:37.000000000 -0800
+++ linux-2.6.4-rc2-42/kernel/panic.c	2004-03-08 12:41:22.297469095 -0800
@@ -96,9 +96,9 @@
 #ifdef __sparc__
 	{
 		extern int stop_a_enabled;
-		/* Make sure the user can actually press L1-A */
+		/* Make sure the user can actually press Stop-A */
 		stop_a_enabled = 1;
-		printk(KERN_EMERG "Press L1-A to return to the boot prom\n");
+		printk(KERN_EMERG "Press Stop-A (L1-A) to return to the boot prom\n");
 	}
 #endif
 #if defined(CONFIG_ARCH_S390)
